### PR TITLE
chore(db-setup): setup sequelize and postgres

### PR DESCRIPTION
**Pull request by ChuckD30**

### CODE REVIEW

File: .sequelizerc [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const path = require('path')
+
+module.exports = {
+  config: path.resolve('./database/config', 'config.js'),
+  'models-path': path.resolve('./database/models'),
+  'seeders-path': path.resolve('./database/seeders'),
+  'migrations-path': path.resolve('./database/migrations'),
+}
```
Comment (codinger41) [inline .sequelizerc]: EOL; Always add one line at the end of each file.

File: database/config/config.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  },
+  test: {
+    url: process.env.TEST_DATABASE_URL,
+    dialect: 'postgres',
+  },
+  production: {
+    url: process.env.DATABASE_URL,
+    dialect: 'postgres',
+  },
+}
```
Comment (codinger41) [inline database/config/config.js]: EOL here too. Just add a one-line free space.

File: .sequelizerc (hunk 1 above)
Comment (ChuckD30) [reply .sequelizerc]: noted.

File: database/config/config.js (hunk 2 above)
Comment (ChuckD30) [reply database/config/config.js]: noted.

**Pull request merged**